.node-version

```diff
@@ -0,0 +1 @@
+6
```

Comment:
Won't matter here, but keep in mind that in most repos, we're going to set this to the most specific version we can to ensure it doesn't upgrade on us.